client/src/components/Header/Header.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styled from "@emotion/styled";
+import logoSrc from "./logo.svg";
+import menuSrc from "./menu.svg";
+
+const Header = () => {
+  return (
+    <Container>
+      <MenuButton>
+        <img src={menuSrc} alt="Menu icon" />
+      </MenuButton>
+      <Logo src={logoSrc} alt="MarryMe logo" />
+    </Container>
+  );
+};
+
+export default Header;
+
+//Styling
+const Container = styled.div`
```

Comment:
the comment can be deleted here :) It's useless, because you can see that the styling is beginning here